collectors/charts.d.plugin/apcupsd/apcupsd.chart.sh

```diff
@@ -129,53 +132,60 @@ BEGIN {
 	time = 0;
 }
 /^BCHARGE.*/		{ battery_charge = \$3 * 100 };
-/^BATTV.*/		{ battery_voltage = \$3 * 100 };
+/^BATTV.*/		    { battery_voltage = \$3 * 100 };
```

Comment:
wrong indents

Reply:
Tabs don't do it for these cases. Converted all to spaces.